index.html

```diff
@@ -289,6 +289,16 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+
+      <section id="contact-me">
+        <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+        <form>
+            <input class="contact-me-field" type="text" name="name" maxlength="30" placeholder="Full name" required>
+            <input class="contact-me-field" type="email" name="email" placeholder="Email address" required>
+            <textarea id="contact-me-message" name="message" maxlength="500" placeholder="Enter text here" required></textarea>
+            <button type="submit">Get in touch</button>
```

Comment:
**[OPTIONAL]:** To improve the UX/UI I recommend implementing the button interactions provided by the [template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) you chose :bulb: :art: